### PR TITLE
Integrate Firebase with Organization Banner

### PR DESCRIPTION
**Pull request by IITvamp**
## Description
removing hardcoded data from organization banner and integrating the firebase data for the same

## Related Issue
#436 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by Maahi10001 — DISMISSED** [review]

File: src/components/Organization/ViewOrganization/index.js [hunk 1]
```diff
@@ -48,12 +48,30 @@ const ViewOrganization = () => {
...
+      profile: {
+        data: { organizations }
+      },
+      org: {
+        general: { current }
+      }
+    }) => organizations[0]
+  );
 
+  console.log(CurrentOrg);
```
Comment (Niweera) [inline src/components/Organization/ViewOrganization/index.js:71]: remove this console.log

Comment (Niweera) [reply src/components/Organization/ViewOrganization/index.js:71]: check other files too, they have console.log functions as well

Comment (IITvamp) [reply src/components/Organization/ViewOrganization/index.js]: done👍

File: src/components/Organization/pages/General.js [hunk 2]
```diff
@@ -147,6 +147,8 @@ function General() {
       }
     }) => organizations.find(element => element.org_handle === current)
   );
+
+  console.log(CurrentOrg);
```
Comment (Niweera) [inline src/components/Organization/pages/General.js:151]: you forgot to remove this.

Comment (IITvamp) [reply src/components/Organization/pages/General.js]: my bad. i have removed it now.

**Review by Niweera — APPROVED** [review]

**Review by Sarfraz-droid — APPROVED** [review]

**Pull request merged**